### PR TITLE
added review form 

### PR DESCRIPTION
**Pull request by prasadsourya**
Please review my work

### CODE REVIEW

File: project1/application.py [hunk 1]
```diff
@@ -95,26 +95,40 @@ def user():
...
+    if (request.method=="POST"):
+        rating=request.form.get("rating")
+        comment=request.form.get("comment")
+        unique=isbn + "_" +session["username"]
+        try:
+            reviews=Review(isbn=isbn,username=session["username"],rating=comment,review=rating,isbnusername=unique)
+            db.session.add(reviews)
+            db.session.commit()
+        except:
+            return "A user can review a single book only once"
```
Comment (madhulika9293) [inline project1/application.py]: Returning text is not good practice. Use html pages only.

File: project1/application.py [hunk 2]
```diff
@@ -95,26 +95,40 @@ def user():
...
+        print(review1) 
+        return render_template("book.html",isbn=isbn,title=book[0].title,author=book[0].author,pub_year=book[0].pub_year,review1=review1)
+    if (request.method=="POST"):
+        rating=request.form.get("rating")
+        comment=request.form.get("comment")
+        unique=isbn + "_" +session["username"]
+        try:
+            reviews=Review(isbn=isbn,username=session["username"],rating=comment,review=rating,isbnusername=unique)
+            db.session.add(reviews)
+            db.session.commit()
```
Comment (madhulika9293) [inline project1/application.py]: Good job with this!

File: project1/templates/fun.html [hunk 3]
```diff
@@ -0,0 +1,393 @@
+<!DOCTYPE html>
```
Comment (madhulika9293) [inline project1/templates/fun.html:1]: Please explain what this page is for.

**Review by madhulika9293 — COMMENTED** [review]
Good job! I suggested one minor change. You can merge after that.

File: project1/templates/fun.html (hunk 3 above)
Comment (prasadsourya) [reply project1/templates/fun.html:1]: It's just a html of a car moving

File: project1/application.py (hunk 2 above)
Comment (prasadsourya) [reply project1/application.py]: thanks

File: project1/application.py (hunk 1 above)
Comment (prasadsourya) [reply project1/application.py]: ok

**Pull request merged**